src/app/containers/PageViewAnalytics/labelHelpers/article.js

```diff
@@ -49,3 +49,14 @@ export const getThingAttributes = (attribute, articleData) => {
 
   return null;
 };
+
+export const getArticleHomeService = articleData => {
```

Comment:
Since this helper isn't needed for analytics, it'd be better to move this helper out of this Analytics folder and into a new file. e.g. `getPassportHome.js` in this utility directory: https://github.com/bbc/simorgh/tree/latest/src/app/lib/utilities
```suggestion
export const getPassportHome = pageData => {
```
The reason I suggest calling it getPassportHome is that it explains where `home` is from - the Passport given by the Content Management System. 

The reason for changing the argument to pageData is that there'll be multiple page types with passports in future, not just articles.